### DISCUSSION TITLE
Default port on InfluxDB hook

### DISCUSSION THREAD

**Discussion by malwini** [apache/airflow#29718] [Q&A]
Hey all!

We have been looking at the [influxdb hook](https://github.com/apache/airflow/blob/main/airflow/providers/influxdb/hooks/influxdb.py) for airflow and we noticed that the [default port used](https://github.com/apache/airflow/blob/78115c5ca995be6f0e06f68b4d97f7c270c16685/airflow/providers/influxdb/hooks/influxdb.py#L70) for the connection was 7686. We can't find any reference to this port in the influx documentation. It seems that the default port for the influx HTTP API is rather 8086: https://docs.influxdata.com/influxdb/v1.8/administration/ports/#8086

Does someone know where 7686 is coming from?

Cheers!

**Comment by potiuk** [comment]
It's `7687` to be precise. Personally I have no idea but maybe @subkanthi would know - he implemented the original PR with  the port hard-coded it seems  https://github.com/apache/airflow/pull/17068
